tests/functional/api/test_clusters.py

```diff
@@ -1,4 +1,10 @@
-def test_project_clusters(project):
+import pytest
+
+
+def test_project_clusters(project, gitlab_version):
+    if gitlab_version.major >= 15:
+        # https://gitlab.com/groups/gitlab-org/configure/-/epics/8
+        pytest.skip("Project clusters are deprecated")
```

Comment:
Should we maybe remove these cluster (group + project) tests entirely? The cert-based integration was already deprecated in 14.5 so we probably don't need to "officially" test this anymore, since we don't run tests against older versions.

Reply:
Done